build-support/bin/ci.sh

```diff
@@ -225,6 +227,26 @@ if [[ "${skip_contrib:-false}" == "false" ]]; then
   end_travis_section
 fi
 
+if [[ "${skip_rust_tests:-false}" == "false" ]]; then
+  source ${REPO_ROOT}/build-support/bin/native/bootstrap.sh
+  ensure_native_build_prerequisites
```

Comment:
This is fine but it may be worth circling back later to `bootstrap_native_code` in `bootstrap.sh` and plumbing an optional flag argument to gate bootstrapping on running tests. Basically keeping knowledge of where rustup is, etc encapsulated there seems better.

Reply:
Did a little tidy-up.